pkg/sfu/receiver.go

```diff
@@ -35,7 +35,6 @@ type TrackReceiver interface {
 	Codec() webrtc.RTPCodecCapability
 
 	ReadRTP(buf []byte, layer uint8, sn uint16) (int, error)
-	GetSenderReportTime(layer int32) (rtpTS uint32, ntpTS buffer.NtpTime)
```

Comment:
Removing this method as this was by down track to get sender report time of currently forwarded layer which is incorrect as the time stamp of each layer is independent. If we go back to locking timestamp of down track to timestamp of specific layer, we can re-visit this. For now, down track uses its timestamp only for sender report. So, this is unnecessary.